### PR TITLE
Add workflow_dispatch trigger for release

### PR DESCRIPTION
**Pull request by louwers**
I will trigger this from the `maplibre-native` repo to make an automated release.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -5,14 +5,14 @@ let package = Package(
...
-            targets: ["Mapbox"])
+            name: "MapLibre",
+            targets: ["MapLibre"])
     ],
     dependencies: [
     ],    
     targets: [
         .binaryTarget(
-            name: "Mapbox",
+            name: "MapLibre",
```
Comment (HarelM) [inline Package.swift:15]: The line below still has mapbox in it...

**Review by HarelM — APPROVED** [review]
There's still a zip file which is called mapbox.zip...

File: Package.swift (hunk 1 above)
Comment (louwers) [reply Package.swift:15]: Yes but that will be auto-replaced when a release is made.

**Comment by louwers** [comment]
@HarelM Yes that is fine. The workflow will replace it.

**Comment by HarelM** [comment]
I mean this file:
`https://github.com/maplibre/maplibre-native/releases/download/ios-v5.13.0/Mapbox-5.13.0.zip`

**Comment by roblabs** [comment]
While testing the release from this PR using the Swift Playground, MapLibreTest.playground, and inspecting the downloaded binary, I have some feedback items:

* Reproduction steps:  Clone this repo, then do `swift package resolve` and inspect the `.build` folder.
* Yes, the packaging name changes in the `.build` folder as expected.
* But the version number, `6.0.0` is not kept upto date.  That isn't set by this repo, but by the native project:  [search for 5.9.0](https://github.com/search?q=repo%3Amaplibre%2Fmaplibre-native%205.9.0&type=code).  I would say this is not a blocker, but any users of the iOS version API would have a wrong SemVer reported.

---

image

**Comment by roblabs** [comment]
The Swift Playground, MapLibreTest.playground, will also need some maintenance as it does not work out of the box.  The goal is to have a properly working Swift Playground when distributing the app, thereby giving iOS developers a known working example how to `import MapLibre`.

I should also note the file size for `6.0.0`'s ZIP decreased to 8.7 MB from 19 MB (using `wget --spider`).  So good job on that minor improvement on the build process.

**Comment by louwers** [comment]
Thanks for trying it out! I will work on those.

> I should also note the file size for 6.0.0's ZIP decreased to 8.7 MB from 19 MB (using wget --spider). So good job on that minor improvement on the build process.

Wow! That's a nice improvement indeed.

**Pull request merged**